docs/swagger/spec.yml

```diff
@@ -20,8 +20,12 @@ consumes:
 tags:
```

Comment:
`get /images/` is missing the following filter query parameters:
"minRawDataBytes", "maxRawDataButes", "minResolution", "maxResolution"